test/parallel/test-cluster-send-deadlock.js

```diff
@@ -23,51 +23,51 @@
 // Testing mutual send of handles: from master to worker, and from worker to
 // master.
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const cluster = require('cluster');
 const net = require('net');
 
 if (cluster.isMaster) {
   const worker = cluster.fork();
-  worker.on('exit', function(code, signal) {
+  worker.on('exit', common.mustCall((code, signal) => {
     assert.strictEqual(code, 0, `Worker exited with an error code: ${code}`);
     assert(!signal, `Worker exited by a signal: ${signal}`);
     server.close();
-  });
+  }));
 
-  const server = net.createServer(function(socket) {
+  const server = net.createServer((socket) => {
     worker.send('handle', socket);
   });
 
-  server.listen(0, function() {
+  server.listen(0, () => {
     worker.send({ message: 'listen', port: server.address().port });
   });
 } else {
-  process.on('message', function(msg, handle) {
+  process.on('message', common.mustCall((msg, handle) => {
```

Comment:
@thefourtheye Is it a good idea to wrap this function inside a `common.mustCall`? 

I have a hunch that it _is_ because it contains an assertion. But I'm cautious because there are two paths inside the function, only one of which contains an assertion.